src/indev/lv_indev.c

```diff
@@ -1314,6 +1328,36 @@ static void indev_proc_release(lv_indev_t * indev)
     }
 }
 
+static void indev_proc_pointer_diff(lv_indev_t * indev)
+{
+    lv_obj_t * obj = indev->pointer.last_pressed;
```

Comment:
`    lv_obj_t * obj = indev->pointer.last_pressed;`
Should the user manually click the screen when a new page is loaded, in order to make the rotary works by default?

Reply:
How about searching a scrollable obj instead if `last_pressed` is NULL?